LibVLCSharp.Gtk/VideoView.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using Gdk;
+using Gtk;
+using LibVLCSharp.Shared;
+using Object = Gtk.Object;
+
+namespace LibVLCSharp.Gtk
+{
+    public class VideoView : DrawingArea, IVideoView
+    {
+        private MediaPlayer _mediaPlayer;
+
+        public VideoView(MediaPlayer mediaPlayer = null)
+        {
+            this._mediaPlayer = mediaPlayer;
+            Color black = Color.Zero;
+            Color.Parse("black", ref black);
+            this.ModifyBg(StateType.Normal, black);
+
+            this.Realized += (s, e) => { this.Attach(); };
+        }
+
+        public MediaPlayer MediaPlayer
+        {
+            get
+            {
+                return this._mediaPlayer;
+            }
+            set
+            {
+                if (Object.ReferenceEquals(this._mediaPlayer, value))
+                {
+                    return;
+                }
+
+                this.Detach();
+                this._mediaPlayer = value;
+                this.Attach();
+            }
+        }
+
+        void Attach()
+        {
+            if (!this.IsRealized || this._mediaPlayer == null)
+            {
+                return;
+            }
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Do we need to perform older checks for net40 targets here?

Reply:
Xamarin.Forms.Platform.GTK is compatible with 4.5. Shall we lower the version?

Reply:
Let's use 4.7.1 for now, and see if someone is interested.